verification/correlation/baseline-report.md

```
@@ -1,20 +1,20 @@
-- Variants tested: 12
-- Issue kind recall: 91.7% (11/12)
-- Issue kind precision: 61.1% (11/18)
-- Snapshot timestamp: 2026-06-28T10:50:38+09:00
+- Verdict accuracy: 100.0% (14/14)
+- Pairs tested: 5
+- Variants tested: 14
+- Issue kind recall: 92.3% (12/13)
+- Issue kind precision: 63.2% (12/19)
+- Snapshot timestamp: 2026-07-28T09:47:35-07:00
 
 ## Data Table
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)